.github/CODEOWNERS

```diff
@@ -48,39 +53,52 @@ script/github-actions/daily-product-scan/ @department-of-veterans-affairs/qa-sta
 
 src/applications/letters @department-of-veterans-affairs/benefits-team-1-frontend @department-of-veterans-affairs/vsa-authd-exp-frontend
 src/applications/disability-benefits @department-of-veterans-affairs/benefits-team-1-frontend @department-of-veterans-affairs/vsa-authd-exp-frontend @department-of-veterans-affairs/DBEX-TREX
 src/applications/claims-status @department-of-veterans-affairs/benefits-team-1-frontend
 src/applications/appeals @department-of-veterans-affairs/benefits-team-1-frontend @department-of-veterans-affairs/vsa-authd-exp-frontend
 src/applications/sah @department-of-veterans-affairs/benefits-team-1-frontend
+src/applications/static-pages/coe-access @department-of-veterans-affairs/benefits-team-1-frontend
+src/applications/static-pages/nod-cta @department-of-veterans-affairs/benefits-team-1-frontend
+src/applications/static-pages/supplemental-claim @department-of-veterans-affairs/benefits-team-1-frontend
 
 # Benefits Non-Disability
+
 src/applications/vre @department-of-veterans-affairs/benefits-non-disability
 src/applications/lgy @department-of-veterans-affairs/benefits-non-disability
 src/applications/burials @department-of-veterans-affairs/benefits-non-disability
 src/applications/pensions @department-of-veterans-affairs/benefits-non-disability
+src/applications/static-pages/vre-chapter-31 @department-of-veterans-affairs/benefits-non-disability
+src/applications/static-pages/vre-chapter-36 @department-of-veterans-affairs/benefits-non-disability
 
 # VSA Debt
 
 src/applications/disability-benefits/2346 @department-of-veterans-affairs/va-cto-health-products
 src/applications/debt-letters @department-of-veterans-affairs/vsa-debt-frontend @department-of-veterans-affairs/benefits-team-1-frontend @department-of-veterans-affairs/1010-health-apps-frontend
 src/applications/financial-status-report @department-of-veterans-affairs/vsa-debt-frontend
+src/applications/static-pages/manage-va-debt @department-of-veterans-affairs/vsa-debt-frontend
 src/applications/medical-copays @department-of-veterans-affairs/vsa-debt-frontend
+src/applications/static-pages/medical-copays-cta @department-of-veterans-affairs/vsa-debt-frontend
 src/applications/combined-debt-portal @department-of-veterans-affairs/vsa-debt-frontend
 src/applications/static-pages/burial-how-do-i-apply-widget @department-of-veterans-affairs/benefits-ce-nondisability
 
 # Public Websites
 
 src/applications/proxy-rewrite @department-of-veterans-affairs/vsa-public-websites-frontend @department-of-veterans-affairs/va-platform-cop-frontend
 src/platform/site-wide @department-of-veterans-affairs/vsa-public-websites-frontend @department-of-veterans-affairs/va-platform-cop-frontend
-src/applications/static-pages @department-of-veterans-affairs/vsa-public-websites-frontend @department-of-veterans-affairs/va-platform-cop-frontend
+src/platform/landing-pages @department-of-veterans-affairs/vsa-public-websites-frontend @department-of-veterans-affairs/vsa-facilities-frontend
```

Comment:
Not certain about replacing FE with Facilities here vs. adding Facilities on?

Reply:
Leaving this here for Facilities folks to review.

Reply:
@maxx1128 @eselkin ^

Reply:
I think it's better to include the FE team here for public websites since it is code that focuses mainly on the front end.

Reply:
@maxx1128 Do you mean removing Facilities from this?

Reply:
@randimays I was thinking more of adding Facilities in addition to FE.